tests/model/test_model.py

```diff
@@ -3255,11 +3262,11 @@ def test__handle_typing_event(
             ),
         ],
         ids=[
-            "remove_18_in_home_view:already_unmuted:ZFLNone",
-            "remove_19_in_home_view:muted:ZFLNone",
-            "add_19_in_home_view:already_muted:ZFLNone",
-            "add_30_in_home_view:unmuted:ZFLNone",
-            "remove_30_is_muted:already_unmutedZFL139",
+            "remove_18_in_home_view:already_unmuted:ZFL0",
+            "remove_19_in_home_view:muted:ZFL0",
+            "add_19_in_home_view:already_muted:ZFL0",
+            "add_30_in_home_view:unmuted:ZFL0",
+            "remove_30_is_muted:already_unmuted:ZFL139",
```

Comment:
The absence of a ":" after already_unmuted seemed inconsistent with other ids. 
While this change is unrelated to the commit/PR, it appears too minor to be a separate commit. Is that fine?